packages/@aws-cdk/aws-certificatemanager/test/example.dns-validated-request.lit.ts

```diff
@@ -1,26 +0,0 @@
-import * as route53 from '@aws-cdk/aws-route53';
```

Comment:
Can you un-delete this file, given we are not (yet) deprecating the DnsValidatedCertificate?